packages/language-server/src/__test__/completion.test.ts

```diff
@@ -721,6 +704,209 @@ suite('Completions', function () {
         },
       })
     })
+    test('Diagnoses type suggestions in model - SQLite, PostgreSQL, MySQL, SQL Server, CockroachDB', () => {
+      for (const provider of ['sqlite', 'postgresql', 'mysql', 'sqlserver', 'cockroachdb']) {
+        console.info(`provider = ${provider}`)
+        assertCompletion({
+          provider: provider as DatasourceProvider,
+          schema: /* Prisma */ `
+            model Post {
+              something |
+            }
+            enum PostType {
+              ADMIN
+              NORMAL
+            }
+            model Something {
+              id Int @id
+            }
+          `,
+          expected: {
+            isIncomplete: true,
+            items: [
+              { label: 'String', kind: CompletionItemKind.TypeParameter },
+              { label: 'Boolean', kind: CompletionItemKind.TypeParameter },
+              { label: 'Int', kind: CompletionItemKind.TypeParameter },
+              { label: 'Float', kind: CompletionItemKind.TypeParameter },
+              { label: 'DateTime', kind: CompletionItemKind.TypeParameter },
+              { label: 'Json', kind: CompletionItemKind.TypeParameter },
+              { label: 'Bytes', kind: CompletionItemKind.TypeParameter },
+              { label: 'Decimal', kind: CompletionItemKind.TypeParameter },
+              { label: 'BigInt', kind: CompletionItemKind.TypeParameter },
+              {
+                label: 'Unsupported',
+                kind: CompletionItemKind.TypeParameter,
+              },
+              { label: 'Post', kind: CompletionItemKind.Reference },
+              { label: 'PostType', kind: CompletionItemKind.Reference },
+              { label: 'Something', kind: CompletionItemKind.Reference },
+            ],
+          },
+        })
+      }
+    })
+  })
+
+  suite('NATIVE TYPES', () => {
+    suite('CockroachDB', () => {
```

Comment:
We should duplicate this for PostgreSQL and see that it suggest _differnet_ things here.